packages/app/src/app/overmind/effects/live/index.ts

```diff
@@ -45,13 +45,16 @@ declare global {
   }
 }
 
+const TIME_TO_THROTTLE_SOLO_MODE_SENDS = 2000;
+
 class Live {
   private identifier = uuid.v4();
   private pendingMessages = new Map();
   private debug = _debug('cs:socket');
   private channel: Channel | null;
   private messageIndex = 0;
   private clients: ReturnType<typeof clientsFactory>;
+  private awaitSendTimer: NodeJS.Timer;
```

Comment:
You can make this

```suggestion
  private awaitSendTimer: number;
```

if you use `window.setTimeout` instead of `setTimeout`. Slightly more correct!